NSS-Spontaneous-Spoonbills/NSS-Spontaneous-Spoonbills
# Created Superuser

## Link to Ticket
No ticket to close

## Description of Proposed Changes
1. Created a superuser 'admin'
1. Updated bangazonapi/admin.py to import all models and register them on admin.

## Steps to Test
1. Setup your virtual machine
1.  Checkout to featured branch
1. You may have to makemigrations  & migrate models
1. Login as admin user 
1. Check to see that all tables are listed
1. Updated the README.md with changes

## Login Information
```
username: admin
password: password123
```

## Check It Out
```
git fetch --all
git checkout JS_django_creating_superuser
```

## Impacted Areas in Application
files: bangazon/admin.py
List general components of the application that this PR will affect:
1. This will affect the admin.py file and give us access to the models via the admin interface.

### Tag users that need to review this code
@MrErin 
@obscure-shadow 

1. The project must be fully documented. This includes the following:
   1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
   1. Every class must be documented with purpose, author, and methods.
   1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
1. Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each
